charts/gitprime-app/v2.0.0/questions.yml

```diff
@@ -59,12 +59,9 @@ questions:
         description: "The directory on the node to use for repository storage"
         required: true
         default: "/mnt/repo-storage"
-      - variable: dataPipeline.volumes.podMountDirectory
-        type: string
-        label: "Directory on Pod"
-        description: "The directory inside the pod to mount the volume."
-        required: true
-        default: "/var/repos"
+      - variable: this.is.group.programming!
```

Comment:
Let's see if this comment shows as having a null VendorUser.

Reply:
Here is a comment from a soon to be deleted user.